<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>background_summary_screen</class>
 <widget class="QDialog" name="background_summary_screen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#background_summary_screen{
background-color: #97C7F9}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="no_channel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C5E1FF;
font: 75 36pt &quot;JasmineUPC&quot;;
border-radius: 25px;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>ช่องที่ 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_header_summary_screen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>841</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;JasmineUPC&quot;;
</string>
   </property>
   <property name="text">
    <string>ข้อมูลของยาที่ต้องทาน</string>
   </property>
  </widget>
  <widget class="QLabel" name="question_pill_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>26</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C5E1FF;
font: 75 26pt &quot;JasmineUPC&quot;;
border-radius: 25px;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>ชื่อยา</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="question_amount_pill">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>26</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C5E1FF;
font: 75 26pt &quot;JasmineUPC&quot;;
border-radius: 25px;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>จำนวนยาที่ต้องทาน</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="question_total_pills">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>26</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C5E1FF;
font: 75 26pt &quot;JasmineUPC&quot;;
border-radius: 25px;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>จำนวนยาทั้งหมด</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="question_time_no1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>26</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C5E1FF;
font: 75 26pt &quot;JasmineUPC&quot;;
border-radius: 25px;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>เวลาที่ 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="button_save_pill_summary">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;JasmineUPC&quot;;
background-color:#24BD73;
color: #ffffff;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>บันทึก</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_pill_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>32</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 32pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>แสดงชื่อยา</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_total_pills">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_amount_pill">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_time">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>12.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="button_edit_pill_name">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/newPrefix/icon_edit.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../Pictures/GUI-KLONG-YAA/icon_edit.qrc">:/newPrefix/icon_edit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="button_edit_total_pills">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>260</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/newPrefix/icon_edit.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../Pictures/GUI-KLONG-YAA/icon_edit.qrc">:/newPrefix/icon_edit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="button_edit_pill_amount_pil">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>340</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/newPrefix/icon_edit.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../Pictures/GUI-KLONG-YAA/icon_edit.qrc">:/newPrefix/icon_edit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="button_edit_time">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/newPrefix/icon_edit.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../Pictures/GUI-KLONG-YAA/icon_edit.qrc">:/newPrefix/icon_edit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text_count_total">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>เม็ด</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_count_amount">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>เม็ด</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_unit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JasmineUPC</family>
     <pointsize>34</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 34pt &quot;JasmineUPC&quot;;
color: #070021;
</string>
   </property>
   <property name="text">
    <string>น.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../Pictures/GUI-KLONG-YAA/icon_edit.qrc"/>
 </resources>
 <connections/>
</ui>
